### ISSUE TITLE
Syntax error when compiling for Unity 2019.3.0f1

### ISSUE THREAD

**Issue by TimmermanV** [yasirkula/UnityAssetUsageDetector#18]
`Assets/Plugins/AssetUsageDetector/Editor/AssetUsageDetector.cs(1450,57): error CS1026: ) expected`

This is because of the closing brackets being inside a preprocessor if:
```
#if !UNITY_2019_3_OR_NEWER
#pragma warning disable 0618
							|| typeof( GUIText ).IsAssignableFrom( currType ) ) )
#pragma warning restore 0618
#endif
```

**Comment by yasirkula** [comment]
Thanks for reporting it. I shall fix it when 2019.3 is released so that I can check if there are any other issues at the same time.

**Comment by yasirkula** [comment]
Fixed in https://github.com/yasirkula/UnityAssetUsageDetector/commit/51abec24d7248564eedc55b4789e68991235cd6e
